docs/docs/docusaurus.config.js

```diff
@@ -0,0 +1,117 @@
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+module.exports = {
+  title: 'Murfin+',
+  tagline: 'Documentation',
+  url: 'https://sardjv.github.io',
+  baseUrl: '/murfin_plus/',
```

Comment:
Needs to be this I think

```suggestion
  baseUrl: '/murfin_method/',
```